test_scripts/API/AutoCompleteList/009_ATF_SetGlobalProperties_P_AutoCompleteList_ArrayOfLowerBound.lua

```diff
@@ -0,0 +1,79 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = true, resultCode = "SUCCESS"> to mobile app
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList> string in array is lower bound.
+-- 3. SDL respond <success = true, resultCode = "SUCCESS"> to mobile app
```

Comment:
@Aleksandriys, looks like you missed "Expected result"(mandatory in template) in header.
I guess, at least p3."SDL respond with <success = true, resultCode = "SUCCESS"> to mobile app" Should be in expected result.